doc/source/whatsnew/v0.17.0.txt

```diff
@@ -669,3 +668,4 @@ Bug Fixes
 - Bug in ``PeriodIndex.order`` reset freq (:issue:`10295`)
 - Bug in ``iloc`` allowing memory outside bounds of a Series to be accessed with negative integers (:issue:`10779`)
 - Bug preventing access to the first index when using ``iloc`` with a list containing the appropriate negative integer (:issue:`10547`, :issue:`10779`)
+- Bug where ``io.gbq`` throws ValueError when Bigquery returns zero rows (:issue:`10273`)
```

Comment:
when they use pandas they use ``pd.read_gbq``. using ``Bigquery`` in a release note about this is not obvious to the casual reader. but that is fine.